dnsapi/dns_cx.sh

```diff
@@ -82,7 +82,7 @@ existing_records() {
     return 1
   fi
   count=0
-  seg=$(printf "%s\n" "$response" | _egrep_o "{[^\{]*host\":\"$_sub_domain\"[^\}]*\}")
+  seg=$(printf "%s\n" "$response" | _egrep_o "{[^{]*host\":\"$_sub_domain\"[^}]*}")
```

Comment:
It seems work well on my FreeBSD 11 VM too:
![image](https://cloud.githubusercontent.com/assets/2725379/20864637/300dbdba-ba34-11e6-9bf0-2b5b54aded47.png)

I restore that file, and it failed:
![image](https://cloud.githubusercontent.com/assets/2725379/20864621/94b9df6a-ba33-11e6-8c08-24bef72b6247.png)